### DISCUSSION TITLE
Connector GA: Script error for "rootPath/DesktopModules/Connectors/GoogleAnalytics/scripts/connector"

### DISCUSSION THREAD

**Discussion by wbonekamp** [dnnsoftware/Dnn.Platform#5542] [General]
## Description of bug
Upgraded to 9.10.2 and connector google analytics is empty:

![image](https://user-images.githubusercontent.com/7567981/220094145-d2797348-c578-483f-8d57-dbce6fb9c9b5.png)

Console error:

![image](https://user-images.githubusercontent.com/7567981/220094041-fdc38d28-9546-4e22-9cdc-0792fa0cb110.png)

## Steps to reproduce
List the precise steps to reproduce the bug:
1. Go to 'Settings'
2. Connectors
3. Google Analytics - "Edit"

## Affected version

* [ ] 09.10.2 latest supported release

## Affected browser
Issue only in chrome not in Firefox
* [ ] Chrome

**Comment by valadas** [comment]
At this stage, this issue is not easily reproducible. I have many 9.10.2 instances without this problem. I'll move this issue to a discussion for now.

The BLOCKED_BY_CLIENT errors make me think it may be a local issue. 
Have you tried:
1. Putting your browser in incognito mode to rule out a bad plugin
2. Trying another browser
3. Trying to temporarily disable your anti-virus
4. Trying from another computer

If none of the above helps the issue, we gonna need a few more steps to make it happen and resolve a potential bug.
